src/solver/variable/economy/unsupliedEnergyCsr.h

```diff
@@ -1,53 +1,53 @@
 /*
-** Copyright 2007-2023 RTE
-** Authors: Antares_Simulator Team
-**
-** This file is part of Antares_Simulator.
+** Copyright 2007-2024, RTE (https://www.rte-france.com)
+** See AUTHORS.txt
+** SPDX-License-Identifier: MPL-2.0
+** This file is part of Antares-Simulator,
+** Adequacy and Performance assessment for interconnected energy networks.
 **
 ** Antares_Simulator is free software: you can redistribute it and/or modify
-** it under the terms of the GNU General Public License as published by
-** the Free Software Foundation, either version 3 of the License, or
+** it under the terms of the Mozilla Public Licence 2.0 as published by
+** the Mozilla Foundation, either version 2 of the License, or
 ** (at your option) any later version.
 **
-** There are special exceptions to the terms and conditions of the
-** license as they are applied to this software. View the full text of
-** the exceptions in file COPYING.txt in the directory of this software
-** distribution
-**
 ** Antares_Simulator is distributed in the hope that it will be useful,
 ** but WITHOUT ANY WARRANTY; without even the implied warranty of
 ** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-** GNU General Public License for more details.
-**
-** You should have received a copy of the GNU General Public License
-** along with Antares_Simulator. If not, see <http://www.gnu.org/licenses/>.
+** Mozilla Public Licence 2.0 for more details.
 **
-** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
+** You should have received a copy of the Mozilla Public Licence 2.0
+** along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
 */
-#ifndef __SOLVER_VARIABLE_ECONOMY_SpilledEnergyAfterCSR_H__
-#define __SOLVER_VARIABLE_ECONOMY_SpilledEnergyAfterCSR_H__
+#pragma once
 
 #include "../variable.h"
 
-namespace Antares::Solver::Variable::Economy
+namespace Antares
+{
+namespace Solver
+{
+namespace Variable
 {
-struct VCardSpilledEnergyAfterCSR
+namespace Economy
+{
```

Comment:
`namespace Antares::Solver::Variable::Economy` was replaced with brace-nested namespaces. 
I guess this **8.8.x** code is not supposed to be read again.

Reply:
Yes